### ISSUE TITLE
Allow fast scroll when pagingEnabled=YES

### ISSUE THREAD

**Issue by slotmaster** [nicklockwood/iCarousel#719]
Our scenario:
1) After scrolling stops, the center needs to show full item. It can't show half of an item. This can be achieved by setting pagingEnabled=YES.
2) Flick needs to be smooth and will skip multiple items if the user flicks hard. From your comment it appears pagingEnabled=YES will disable this.

Is there a way to satisfy both? I actually think pagingEnabled=YES shouldn't disable the skipping. If user wants to disable skipping, he can set decelerationRate to 0. decelerationRate and pagingEnabled should work together. Right now pagingEnabled=YES will cause the effect as if decelerationRate=0.

**Comment by nicklockwood** [comment]
The carousel should never stop on half an item anyway. If you check the example projects you'll see that the carousel always comes to a stop with an item in the center of the view (if that's not the case, you've found a previously unknown bug).

I suspect you've got some misalignment issue with your carousel frame that is causing the selected item to be off-center.

**Comment by slotmaster** [comment]
Yes, I found that sometimes it's off-center. Glad to know it's unexpected. I will investigate to see what the root cause is.
